lib/oli_web/templates/project/overview.html.eex

```diff
@@ -1,8 +1,10 @@
-<div class="overview container">
+<div class="overview container" xmlns="http://www.w3.org/1999/html">
```

Comment:
Is this xmlns necessary?  If not, it should be removed.

Reply:
Deleted unnecessary xmlns